docs/release_process.md

```diff
@@ -0,0 +1,62 @@
+# cellxgene-schema Release Process
+
+_This document defines the release process of the pip installable `cellxgene-schema` tool to PyPI._
+
+The goal of the release process is to publish an installable package to PyPi, with a matching tagged release on github.
+
+## Overview
+
+The release process should result in the following side effects:
+
+* Version number bump, using semantic versioning
+* Tagged Github release
+* Publication to PyPi
+
+Note all release tags pushed to GitHub MUST follow semantic versioning.
+
+## How to release the next version of `cellxgene-schema`
+
+Steps must be run from the project directory and in a virtual env with all the dependencies defined in `requirements.txt` installed.
+
+1. Decide on whether the changes going into the next release warrant a patch, minor, or major version bump using [semantic versioning](https://semver.org/) and determine the version number based on the current latest version (i.e. if the current version is 2.0.4 and you'd like to do a minor version bump, then the next version will be 2.1.0).
+
+1. Create a release branch from the `main` branch of the `single-cell-curation` repo (e.g. `git checkout -b release-version-x.y.z`).
+
+1. Run `make create-release-candidate PART={major | minor | patch}`, where you set `PART` to be one of `major` or `minor` or `patch` based on Step 1. This step will automatically bump the version of the package to the appropriate next version in `setup.py` and commit the change to the branch. Note that this will create a release candidate version, so if the current version is 2.0.4 and you want to do a minor release, then the version will be 2.1.0rc0. The `rc` suffix will be removed in a later step.
```

Comment:
```suggestion
1. Run `make create-release-candidate PART={major | minor | patch}`, where you set `PART` to be one of `major` or `minor` or `patch` based on Step 1. This step will automatically bump the version of the package to the appropriate next version in `setup.py` and commit the change to the branch. Note that this will create a release candidate version, so if the current version is 2.0.4 and you want to do a minor release, then the version will be `2.1.4-rc.0`. The `rc` suffix will be removed in a later step.
```